### DISCUSSION TITLE
新聞分享：影片資料庫Ego4D

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#71] [General]
**新聞大綱：**
Facebook宣布與13間大學合作，共同打造一款以第一人稱視角累積的影片資料庫Ego4D，藉此讓人工智慧可以透過此類影片內容進行學習，進而知曉以第一人稱視角所見視野內，進而做出人類判斷相近的決策結果。

依照Facebook首席科學家Kristen Grauman說明，由於過去多半是以側錄拍攝，因此難以將人工智慧判斷決策與結果拉近，為了解決問題，Ego4D便以大量第一人稱視角影片內容，讓人工智慧系統進行深度學習。

在蒐集影片內容中，提供頭戴式攝影機，以及感測元件，藉此在紀錄第一人稱視角影片時，同時也能紀錄身體動作資訊。至於紀錄內容包括物件外，也包含與身體活動數據，讓人工智慧理解人類在面臨不同視覺體驗時，身體會做出什麼樣的反應。

目前Ego4D吸引超過700名參與者，以第一人稱視角收集超過2200小時的影片內容，而參與者分別來自英國、義大利、印度、日本、沙烏地阿拉伯…，因此也讓人工智慧能依照人種、年齡與性別等額外差異進行深度學習，進而建立不同學習結果。

**新聞心得：**
我覺得這個新發明的影片資料庫很有意義，不僅讓參與體驗過程的人員能有一個新奇的經驗，也使其中提供的數據讓AI學習人類的反應、做事當下的感受，進而在未來可以使用在更廣泛的應用面！

**新聞連結：**
https://udn.com/news/story/7086/5820263

**Comment by awehao** [comment]
畢竟是人工智慧，想想也確實要讓其用人類的角度觀看世界
